### ISSUE TITLE
McStas 2018 Survey Feedback 16

### ISSUE THREAD

**Issue by willend** [McStasMcXtrace/McCode#741]
3D visualization without compiling needed. Everytime I add a new COMPONENT the window get refreshed.

**Comment by farhi** [comment]
Yes that would be nice. But the 3D view is only obtained after compilation. When do we trigger it ?
When the instr file has not been touched for say 5 sec ?
A background process in mcgui could monitor the typing/save and launch a 3D view.
However, often the initial view angle is not appropriate (e.g. very long models). Not sure this helps in this case.
Let's be objective: users do launch a 3D view when they actually converge to something. this is yet a manual process, but only when assembling the bits and pieces.

**Comment by willend** [comment]
No can do.
